### PR TITLE
Table component

### PR DESCRIPTION
**Pull request by jeremy-babylonlabs**
Screenshot 2024-12-02 at 8 39 47 PM

- Bump version of `@babylonlabs-io/bbn-core-ui` to 0.2.0
- Add new `Table` component with sorting and loading features

### CODE REVIEW

File: tsconfig.node.json [hunk 1]
```diff
@@ -17,8 +16,7 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true,
-    "noUncheckedSideEffectImports": true
```
Comment (gbarkhatov) [inline tsconfig.node.json:21]: Why do we remove it?

File: tsconfig.node.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
```
Comment (gbarkhatov) [inline tsconfig.node.json:3]: Any reason we remove it while introducing the Table component?

File: package.json [hunk 3]
```diff
@@ -52,12 +52,15 @@
     "@storybook/test": "^8.4.2",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
+    "@typescript-eslint/eslint-plugin": "^8.16.0",
```
Comment (gbarkhatov) [inline package.json:55]: Are we using these 3 new plugins ?

**Review by gbarkhatov — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (jeremy-babylonlabs) [reply package.json]: i was playing around with the. configs, forgot to remove it. thanks for the reminder

**Pull request merged**